### DISCUSSION TITLE
Stuttering with A320 stand-alone plane

### DISCUSSION THREAD

**Discussion by Andysax68** [flybywiresim/aircraft#4720] [Help]
Hello team, I experience stuttering with the A320 since it's a stand-alone plane (it wasn´t causing that when it was a mod of the Asobo A320). Here is a screenshot of my MSFS, A320 sitting on Rwy 21 at LECO (no mods or add-on in the area) with clear rhythmic stuttering, possibly caused by glass cockpit refresh (set at slow in Options).
![a320FBW](https://user-images.githubusercontent.com/83730918/117207470-45d2cb00-adf4-11eb-8983-914e7942a76e.png)

**Comment by tracernz** [comment]
Did it start with world update 4? There is a new MSFS patch today that should fix it if so.

**Reply by Andysax68** [reply]
no, it started when I updated the A320 using the new downloader, and it created the new plane totally independent from the Asobo A320.

**Reply by Andysax68** [reply]
Tried the patch, does not change anything

**Reply by ChiChiJim** [reply]
Adding my experience...Running the stable version V 0.6.0. Everything runs fine until final approach, about 100 nm away from the airport. Performance lag is horrible. Dropping frames. I tried reducing my graphics. No improvements.  Auto pilot is flying the whole time.

**Reply by claus737sim** [reply]
Got the same problem with heavy stuttering now in nearly all airports. In VR with even medium settings on an 11900 5Ghz and 3080 10GB with updated drivers it's not fun to fly anymore.

The MS 320 runs smooth also AS CRJ with no problems......
